docs/development/plugins/index.md

```diff
@@ -32,9 +32,14 @@ for more details.
 
 ## Developing Plugins
 
-Plugins are supposed to be built and shipped out-of-tree, i.e. instead of managing the plugins'
+Plugins are supposed to be built and shipped out-of-tree, i.e., instead of managing the plugins'
 code within the Headlamp project or a Headlamp fork (which would require
 always rebuilding/maintaining Headlamp when changing a plugin), they're
 supposed to be built outside of the project and loaded by Headlamp.
 
+Plugins are meant to be build and shipped out-of-tree (i.e., outside of the project and loaded by
```

Comment:
Looks like there's an extra paragraph here.

Reply:
Ah! Thanks for catching this